README.md

```diff
@@ -959,27 +959,10 @@ _You can enable the following settings in Xcode by running [this script](resourc
 
   </details>
 
-* <a id='private-constants'></a>(<a href='#private-constants'>link</a>) **Prefer putting constants in the top level of a file if they are `private`.** If they are `public` or `internal`, define them as static properties, for namespacing purposes.
-
-  <details>
-
-  ```swift
-  private let privateValue = "secret"
-
-  public class MyClass {
-
-    public static let publicValue = "something"
-
-    func doSomething() {
-      print(privateValue)
-      print(MyClass.publicValue)
-    }
-  }
-  ```
-
-  </details>
-
-* <a id='namespace-using-enums'></a>(<a href='#namespace-using-enums'>link</a>) **Use caseless `enum`s for organizing `public` or `internal` constants and functions into namespaces.** Avoid creating non-namespaced global constants and functions. Feel free to nest namespaces where it adds clarity.
+* <a id='namespace-using-enums'></a>(<a href='#namespace-using-enums'>link</a>) **Prefer using caseless `enum`s for organizing `public` or `internal` constants and functions into namespaces.**
+  * Feel free to nest namespaces where it adds clarity.
+  * Avoid creating non-namespaced global constants and functions.
+  * `private` globals are permitted, since they are scoped to a single file and do not pollute the global namespace. Consider placing private globals in an enum namespace to match the guidelines for other declaration types.
```

Comment:
```suggestion
  * `private` globals are permitted, since they are scoped to a single file and do not pollute the global namespace. Consider placing private globals in an `enum` namespace to match the guidelines for other declaration types.
```

... to match the earlier part of the rule